Yoast/wordpress-seo
# JSON_UNESCAPED_SLASHES flag added to prevent unnecessary slashes from…

… schema.org JSON data

## Summary

This PR can be summarized in the following changelog entry:

* Looks like using built-in WP "wp_json_encode()" function without any option causes the issue  #10619 .

## Relevant technical choices:

* It uses PHP's native "json_encode()" function which creates slashes, when used without "JSON_UNESCAPED_SLASHES" flag.

## Test instructions

This PR can be tested by following these steps:

* Open the source code of homepage and see the "schema.org" JSON data before and after this commit.

## Quality assurance


Fixes #
